plugins/optimization-detective/helper.php

```diff
@@ -37,18 +37,18 @@ function od_initialize_extensions(): void {
  *
  * @since 0.7.0
  *
- * @param int|null $minimum_viewport_width Minimum viewport width.
- * @param int|null $maximum_viewport_width Maximum viewport width.
+ * @param int<0, max>|null $minimum_viewport_width Minimum viewport width (exclusive).
+ * @param int<1, max>|null $maximum_viewport_width Maximum viewport width (inclusive).
  * @return non-empty-string|null Media query, or null if the min/max were both unspecified or invalid.
  */
 function od_generate_media_query( ?int $minimum_viewport_width, ?int $maximum_viewport_width ): ?string {
-	if ( is_int( $minimum_viewport_width ) && is_int( $maximum_viewport_width ) && $minimum_viewport_width > $maximum_viewport_width ) {
-		_doing_it_wrong( __FUNCTION__, esc_html__( 'The minimum width cannot be greater than the maximum width.', 'optimization-detective' ), 'Optimization Detective 0.7.0' );
+	if ( is_int( $minimum_viewport_width ) && is_int( $maximum_viewport_width ) && $minimum_viewport_width >= $maximum_viewport_width ) {
+		_doing_it_wrong( __FUNCTION__, esc_html__( 'The minimum width cannot be greater than or equal to the maximum width.', 'optimization-detective' ), 'Optimization Detective 0.7.0' );
 		return null;
 	}
 	$media_attributes = array();
 	if ( null !== $minimum_viewport_width && $minimum_viewport_width > 0 ) {
-		$media_attributes[] = sprintf( '(min-width: %dpx)', $minimum_viewport_width );
+		$media_attributes[] = sprintf( '(min-width: %dpx)', $minimum_viewport_width + 1 );
```

Comment:
The addition of 1 here will can be undone as part of https://github.com/WordPress/performance/pull/1833. /cc @SohamPatel46

Reply:
On the issue you mentioned potential issues with half pixels. By no means a blocker, but I would think that doing this here still leaves that problem unresolved for the largest viewport's minimum width.

Is there no way to use an exclusive minimum without providing a maximum with the new range syntax?

Reply:
Looks like you can: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_media_queries/Using_media_queries#:~:text=%40media%20(width%20%3C%3D%201250px)%20%7B

Reply:
Yes, but I figured to do that as part of https://github.com/WordPress/performance/pull/1833 since doing that change here would cause all of the snapshots to be invalidated.

Reply:
See https://github.com/WordPress/performance/pull/1833/files#r1939991349